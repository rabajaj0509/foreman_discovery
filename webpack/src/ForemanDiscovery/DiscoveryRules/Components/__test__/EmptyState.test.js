import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
import ForemanEmptyState from '../EmptyState';

const action = {
  title: 'action-title',
  url: `action-url`,
};

const fixtures = {
  'render with Props': {
    docUrl: 'https://foreman.example.com',
    action,
  },
};

describe('ForemanEmptyState', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(ForemanEmptyState, fixtures));
});
