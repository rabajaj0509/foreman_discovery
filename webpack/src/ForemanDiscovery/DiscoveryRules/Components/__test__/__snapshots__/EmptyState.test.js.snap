// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanEmptyState rendering render with Props 1`] = `
<DefaultEmptyState
  action={
    Object {
      "title": "Create Rule",
      "url": "/discovery_rules/new",
    }
  }
  description="No Discovery Rules found in this context. Create Discovery Rules to perform automated provisioning on Discovered Hosts"
  documentation={
    Object {
      "url": "https://foreman.example.com",
    }
  }
  header="Discovery Rules"
  icon="gavel"
  iconType="fa"
  secondaryActions={Array []}
/>
`;
